inject/tests/resources-inject/pom.xml

```diff
@@ -130,6 +125,11 @@
                     </annotationProcessorPaths>
                 </configuration>
                 <dependencies>
+                    <dependency>
+                        <groupId>io.helidon.inject</groupId>
+                        <artifactId>helidon-inject-processor</artifactId>
+                        <version>${helidon.version}</version>
+                    </dependency>
 				    <dependency>
```

Comment:
nit: spacing.

Reply:
This is caused by the original PR not following Helidon code style and using tabs instead of spaces. Looks good in my IDE, as I have tab spacing set to the same value as the default spacing

Reply:
Fixed to spaces